entrypoint.sh

```diff
@@ -11,9 +11,13 @@ fi
 # Move the supervisord socket to /dev/shm to circumvent
 # https://github.com/Supervisor/supervisor/issues/654
 sed -i "s@/opt/datadog-agent/run/datadog-supervisor.sock@/dev/shm/datadog-supervisor.sock@" ${DD_ETC_ROOT}/supervisor.conf
+# for the status command
+if echo "$DD_ETC_ROOT" | grep -q "/etc/dd-agent"; then
```

Comment:
would `if [ -e /etc/init.d/datadog-agent ]; then` be more robust?

Reply:
sure